userspace/libsinsp/test/events_fspath.ut.cpp

```diff
@@ -196,10 +236,23 @@ class fspath : public sinsp_with_test_input
 
 		va_list args;
 		va_start(args, n);
-		sinsp_evt* evt = add_event_advance_ts_v(increasing_ts(), 1, event_type, n, args);
+		sinsp_evt* evt;
+		switch (event_type)
+		{
+			case PPME_SYSCALL_LINKAT_2_X:
+			case PPME_SYSCALL_SYMLINKAT_X:
+				{
+					add_event_advance_ts(increasing_ts(), 1, PPME_SYSCALL_OPENAT2_E, 2, evt_dirfd, dirfd_path);
+					add_event_advance_ts(increasing_ts(), 1, PPME_SYSCALL_OPENAT2_X, 5, evt_dirfd, evt_dirfd, dirfd_path, open_flags | PPM_O_DIRECTORY, mode);
```

Comment:
We are creating a folder, enforce `PPM_O_DIRECTORY` otherwise our `fdinfo` will be off.